docs/reference/search-connectors/es-connectors-sharepoint.md

```diff
@@ -146,7 +146,7 @@ The following configuration fields are required to set up the connector:
 
     * `collection1`
     * `collection1, collection2`
-
+If you need to parse the root from Share point, just use the url of your sharepoint server in `site`collections`
```

Comment:
```suggestion

If you need to parse the root from SharePoint, just use the url of your SharePoint Server in `site_collections`.
```

Reply:
The removed empty line was needed, otherwise the rendering puts the text together with the previous bullet.